src/SprykerSdk/Spryk/Model/Spryk/Definition/Argument/Callback/GlueProcessorModelInterfaceTargetFilenameCallback.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Callback;
+
+use SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface;
+
+class GlueProcessorModelInterfaceTargetFilenameCallback implements CallbackInterface
+{
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'GlueProcessorModelInterfaceTargetFilename';
+    }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface $argumentCollection
+     * @param mixed|null $value
+     *
+     * @return mixed
+     */
+    public function getValue(ArgumentCollectionInterface $argumentCollection, $value)
+    {
+        $className = $argumentCollection->getArgument('className')->getValue();
+        if (strpos($className, '\\') !== false) {
```

Comment:
You can use `if (strpos($className, '\\'))`

Reply:
And the same about $value type as above.

Reply:
The result of `strpos($className, '\\')` has to be checked strictly because it return `false` only in case the `$needle` is not found, whereas in case it is found in the position `0` it will return `0` and your version of condition will fail.